server/routes/images.go

```diff
@@ -0,0 +1,159 @@
+package routes
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"localdev/main/config"
+	"localdev/main/models"
+	"localdev/main/services"
+	"mime/multipart"
+	"net/http"
+	"text/template"
+	"time"
+
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/log"
+
+	"cloud.google.com/go/datastore"
+	"cloud.google.com/go/storage"
+	vision "cloud.google.com/go/vision/apiv1"
+)
+
+func serveError(ctx context.Context, w http.ResponseWriter, err error) {
+	w.WriteHeader(http.StatusInternalServerError)
+	w.Header().Set("Content-Type", "text/plain")
+	fmt.Fprint(w, "Internal Server Error")
+	log.Errorf(ctx, "%v", err)
+}
+
+var rootTemplate = template.Must(template.New("root").Parse(rootTemplateHTML))
+
+// For temporary purposes, to be deleted later
+const rootTemplateHTML = `
+	<html>
+		<body>
+			<form action="{{.}}" method="POST" enctype="multipart/form-data">
+				Upload File: <input type="file" name="file">
+				<br>
+				<input type="submit" name="submit" value="Submit">
+			</form>
+		</body>
+	</html>
+	`
+
+func ImageUploadPage(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+
+	uploadURL := "/image-uploaded/"
```

Comment:
I guess this will be made configurable at some point, this is the reason why you use a variable, right?

Reply:
I planned adding different callback url for profile images. But idk if we'll need this handler at all, it's pretty much a temporary stub for uploading images